### PR TITLE
Feat/known issues

### PR DESCRIPTION
**Pull request by aliceoq**
#### What is the purpose of this pull request?

To add the Known Issues page.

#### What problem is this solving?

The known issues page was not implemented.

#### How should this be manually tested?

Open the preview, navigate and use the filters in the [known issues page](https://deploy-preview-26--leafy-mooncake-7c2e5e.netlify.app/known-issues). Also, open the individual known issue pages, such as, ["List Purchases" link in the gift list report does not apply filter correctly in Orders Managment](https://deploy-preview-26--leafy-mooncake-7c2e5e.netlify.app/known-issues/list-purchases-link-in-the-gift-list-report-does-not-apply-filter-correctly-in-orders-managment) and compare it to the high fidelity prototype.

There are still some fixes to be made, but they will be made in future PRs.
- ⚠️ The 'order by' dropdown component is not identical to the high fidelity prototype. I will create the new component later.
- ⚠️ The filter does not have 'Select all' options, when the user selects 0 filters it is the same as selecting all.
- ⚠️ The loading can be slow on the first time because it fetches all known issues.
- ⚠️ The heading image is different (will fix this later because I will also fix it in the other documentation pages).
- ⚠️ The order by relevance was not added.

#### Screenshots or example usage

https://github.com/vtexdocs/helpcenter/assets/62757720/94f2330b-b86e-4fa9-8983-c71099801925

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by Pvcunha — APPROVED** [review]
LGTM!

**Comment by ThayllyR** [comment]
Muito bom! ✨

**Review by ThayllyR — APPROVED** [review]
LGTM ✨

File: src/pages/known-issues/[slug].tsx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+                    {serialized.frontmatter?.title}
+                  </Text>
+                  <Box sx={styles.divider}></Box>
+                  <Flex sx={styles.detailedInfo}>
+                    <Flex sx={styles.id}>
+                      <Text>ID: {serialized.frontmatter?.id}</Text>
+                      <Tag>{serialized.frontmatter?.kiStatus}</Tag>
+                    </Flex>
+                    <Text sx={styles.dates}>
+                      <em>Criado em </em>
```
Comment (Pvcunha) [inline src/pages/known-issues/[slug].tsx:118]: Ta em português independente da linguagem selecionada

File: src/pages/known-issues/[slug].tsx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+                      {serialized.frontmatter?.createdAt}
+                      {' • '}
+                      <em>Atualizado em </em>
```
Comment (Pvcunha) [inline src/pages/known-issues/[slug].tsx:121]: Aqui também, está sem localização de idioma

**Review by Pvcunha — CHANGES_REQUESTED** [review]

**Review by Pvcunha — APPROVED** [review]
LGTM!

**Review by ThayllyR — APPROVED** [review]
✅✨

File: src/components/filter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        <FilterIcon />
+        <Text>{intl.formatMessage({ id: 'filter_modal.title' })}</Text>
+        {numberOfFilters > 0 && (
+          <Text sx={styles.numberOfFilters}>{numberOfFilters}</Text>
+        )}
+      </Flex>
+    )
+  }
+
+  const FilterModal = () => {
```
Comment (RobsonOlv) [inline src/components/filter/index.tsx:76]: Talvez seja interessante adicionar o hook  `useClickOutside` aqui:
```
  const modalRef = useRef<HTMLDivElement>()
  useClickOutside(modalRef, () => setIsModalOpen(false))
  ...
  <Box ref={modalRef} sx={styles.container}>
  ...
```
Somente para melhorar a experiência do usuário

**Review by RobsonOlv — APPROVED** [review]
LGTM :rocket:

File: src/components/filter/index.tsx (hunk 3 above)
Comment (aliceoq) [reply src/components/filter/index.tsx:81]: Não usei o useClickOutside, mas coloquei pro clique fora do modal fechar 🫡

**Pull request merged**